app/authorisation.js

```diff
@@ -0,0 +1,22 @@
+const config = require('../config')
+
+function isAuthenticated (req) {
+  if (!req.isAuthenticated() || !req.user) {
+    var error = new Error('unauthenticated')
+    error.status = 401
+    throw error
+  }
+}
+
+function hasRole (req, role) {
```

Comment:
I guess we can just rely on the `req.user` only and leave the `req.isAuthenticated()` check. Otherwise we will need to pass the whole request object.